src/main/java/de/tum/cit/aet/artemis/core/service/telemetry/TelemetryService.java

```diff
@@ -22,35 +22,30 @@ public class TelemetryService {
 
     private final TelemetrySendingService telemetrySendingService;
 
-    @Value("${artemis.telemetry.destination}")
-    private String destination;
+    private final boolean useTelemetry;
 
-    @Value("${artemis.telemetry.enabled}")
-    public boolean useTelemetry;
+    private final boolean sendAdminDetails;
 
-    @Value("${artemis.telemetry.sendAdminDetails}")
-    public boolean sendAdminDetails;
-
-    @Value("${eureka.client.enabled:false}")
-    public boolean eurekaEnabled;
-
-    public TelemetryService(ProfileService profileService, TelemetrySendingService telemetrySendingService) {
+    public TelemetryService(ProfileService profileService, TelemetrySendingService telemetrySendingService, @Value("${artemis.telemetry.enabled}") boolean useTelemetry,
+            @Value("${artemis.telemetry.sendAdminDetails}") boolean sendAdminDetails) {
```

Comment:
Is there a reason for moving the `@Value` away from the variable declaration? I thinks it's way better readable up there instead of in the constructor

Reply:
I agree, but to be able to inject them in the test, we need to use the annotation directly in the constructor